tests/unit/document/test_database_subindices.py

```diff
@@ -0,0 +1,49 @@
+import os
+from docarray import Document, DocumentArray, dataclass
+from docarray.typing import Image, Text
+
+
+@dataclass
+class MyDocument:
+    image: Image
+    paragraph: Text
+
+
+cur_dir = os.path.dirname(os.path.abspath(__file__))
+
+sqlite3_config = {'connection': 'jina4.db', 'table_name': 'test4'}
```

Comment:
can we try to randomize the `table names and connection names? So that we can have different tests? these can be inside the test itself